library/PlaceChainMap.m

```diff
@@ -0,0 +1,1857 @@
+  chainMap[@"707805b4-e834-4c1b-84ec-e33b30d8dfdc"] = @"中原地產";
+  chainMap[@"707fd570-684a-4caa-a26b-65593b7a44cb"] = @"Albert Supermarket";
+  chainMap[@"70a0eefd-f247-4a63-af97-26f13e267d38"] = @"日本城";
+  chainMap[@"7143f8d1-4749-417c-a148-c7b15269f65c"] = @"Bloomingdale's";
+  chainMap[@"71454a95-4a7c-4be9-a831-b98518f9cd6c"] = @"PEP";
+  chainMap[@"714fe331-df45-481c-a3fc-c7ff4ee2c974"] = @"Savers";
+  chainMap[@"71528c3b-6632-4996-96f9-44e6a4452e6b"] = @"CITGO";
+  chainMap[@"718e50a0-a373-4a5e-b050-977ff20ca9dc"] = @"Michaels";
+  chainMap[@"71d64628-7daa-4859-b3ad-d016bbe867a7"] = @"中国电信";
+  chainMap[@"71de90e2-0a55-11e5-88ec-db15d8099f42"] = @"Zagrebačka Banka";
+  chainMap[@"71de9344-0a55-11e5-88ed-3bcda9861b41"] = @"FINA";
+  chainMap[@"71de93c6-0a55-11e5-88ee-fb7040ddc932"] = @"Studenac";
+  chainMap[@"71de9420-0a55-11e5-88ef-67690c5ad9e5"] = @"Hrvatska Pošta";
+  chainMap[@"71de9484-0a55-11e5-88f0-b3b7943bd9c5"] = @"Ina";
+  chainMap[@"71de94e8-0a55-11e5-88f1-2ff2b6280e82"] = @"Tommy";
+  chainMap[@"71de9542-0a55-11e5-88f2-6b72c8aef661"] = @"Dm Drogerie Markt";
+  chainMap[@"71de959c-0a55-11e5-88f3-aff964a094e1"] = @"Merkur 5";
+  chainMap[@"71de9600-0a55-11e5-88f4-cbc62f088a19"] = @"Međimurska Banka";
+  chainMap[@"71de965a-0a55-11e5-88f5-779027f71f29"] = @"Petrol";
+  chainMap[@"71de96be-0a55-11e5-88f6-3f31728b510f"] = @"Konzum";
+  chainMap[@"71de9718-0a55-11e5-88f7-2707865672a4"] = @"Billa";
+  chainMap[@"72009bc1-a04c-4f23-b50c-f9c4d627665c"] = @"許留山";
+  chainMap[@"7268dd3e-8c4d-4cad-8813-ba5b6606441d"] = @"Canadian Pizza";
+  chainMap[@"7382e706-6fe6-4e29-83fe-e6aa07e28dc4"] = @"悟饕池上飯包";
+  chainMap[@"73e20d01-98ae-4958-b0fa-e89df2e1c236"] = @"Bank Central Asia";
+  chainMap[@"73f15e4f-ce24-4468-8659-cfd22ea81377"] = @"Tesla Store";
+  chainMap[@"743993d2-344c-4c12-b169-495e0a98a796"] = @"AAMCO Transmissions and Total Car Care";
+  chainMap[@"7465295b-264f-4988-9674-f294d32596fa"] = @"Rustan's";
+  chainMap[@"7473469b-b5ca-4a57-a642-ed2fc25eb847"] = @"Minipreço";
+  chainMap[@"74cb8d61-0ce0-498d-95bf-49d00573eb38"] = @"Siwa";
+  chainMap[@"74cb9556-38d0-43cb-8505-ef4c2e769d75"] = @"Famous Footwear";
+  chainMap[@"74d815be-fba0-4db0-a7b1-5f6cfb227929"] = @"The Body Shop";
+  chainMap[@"7558fa4d-3927-493d-a185-e172a097ecb5"] = @"Albaik";
+  chainMap[@"757e42da-25fd-47c4-aa65-7ba9cb3aa66e"] = @"大家樂";
+  chainMap[@"75e65297-d151-48ff-8095-bbd3f9ff39bb"] = @"Bank of America ATM";
+  chainMap[@"7717a333-98a4-4021-a93e-28e2c25131e3"] = @"Steak 'n Shake";
+  chainMap[@"77c4d528-e157-42be-a523-ff016d396014"] = @"dd's DISCOUNTS";
+  chainMap[@"77dac7d9-08ce-4e89-827a-20d83da749f1"] = @"MOL";
+  chainMap[@"7818ddec-131e-4a23-88ae-048031b32fea"] = @"National Car Rental";
+  chainMap[@"78310576-ab68-4f8b-99bc-21439a5a57eb"] = @"California Pizza Kitchen";
+  chainMap[@"7884ad8b-caa1-4b5a-89a9-3b7456c850ad"] = @"Posti";
+  chainMap[@"790165ed-0a7f-4736-b28b-dc4ffb5e435e"] = @"Sizeer
```

Comment:
`FactualPlace chainId` is defined as a nullable `NSString*`. Therefore, if it is ever set to `[NSNull null]`, we're doing something wrong. Normally, the comparison here should be `!= nil`.